src/pages/Home/TaskCard/AddTaskCard.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
+import style from "./index.module.scss";
+import { sanatizeText } from "../../../utils/helpers/sanatizeText";
+import { useDispatch } from "react-redux";
+import { addTask, createButton } from "../../../store/actions";
+import { v1 as uuidv1 } from "uuid";
```

Comment:
Could we please move the imported things from packages to the top of the file?
```suggestion
import React, { useState } from "react";
import { useDispatch } from "react-redux";
import { v1 as uuidv1 } from "uuid";
import style from "./index.module.scss";
import { sanatizeText } from "../../../utils/helpers/sanatizeText";
import { addTask, createButton } from "../../../store/actions";
```